cli/schema_migration_down.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2023 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package cli
+
+import (
+	"encoding/json"
+
+	"github.com/sourcenetwork/immutable/enumerable"
+	"github.com/spf13/cobra"
+
+	"github.com/sourcenetwork/defradb/client"
+	"github.com/sourcenetwork/defradb/datastore"
+)
+
+func MakeSchemaMigrationDownCommand() *cobra.Command {
+	var schemaVersionID string
+	var cmd = &cobra.Command{
+		Use:   "down --version <version> <src>",
+		Short: "Reverse a migration on the specified schema version.",
```

Comment:
todo: Please explain to the user what `<src>` is :)

suggestion: `on` is somewhat ambiguous here I think, `from` (or `to`, I cant remember which way round) might be better.
 
 Same goes for the up command.

Reply:
I tried to describe it in more detail. I believe it is the list of documents to apply / reverse the migration to / from.

Reply:
cheers Keenan - looks good